perceval/algorithm/tomography/tomography.py

```diff
@@ -239,17 +219,16 @@ class ProcessTomography(AAlgorithm):
         -- Fidelity of the operation, Error process map
 
     """
-    def __init__(self, nqubit: int, operator_processor: Processor, post_process=False,
-                 renormalization=None):
+    def __init__(self, operator_processor: Processor):
         super().__init__(processor=operator_processor)
-        self._nqubit = nqubit
         self._operator_processor = operator_processor
-        self._backend = operator_processor.backend  # default - SLOSBackend()
-        self._post_process = post_process
-        self._renormalization = renormalization
-        self._size_hilbert = 2 ** nqubit
-        self._qst = StateTomography(nqubit=self._nqubit, operator_processor=self._operator_processor,
-                                    post_process=self._post_process, renormalization=self._renormalization)
+        self._nqubit = int(operator_processor.m / 2)  # nqubit
+        self._size_hilbert = 2 ** self._nqubit
+        self._qst = StateTomography(operator_processor=self._operator_processor)
+
+        self.chi_normalized = None
+        self.chi_unnormalized = None
```

Comment:
Why do we want to have the unormalized chi as a class member if we use it only in one method ?

Reply:
This was so that the method doesnot return multiple things but the class attribute gets updated once chi is computed; this will be given to a user only if they ask for it.

I can look up for cleaner/more logical solutions to achieve this.